mlem/ext.py

```diff
@@ -59,9 +59,12 @@ def __hash__(self):
 
     @property
     def reqs_packages(self):
-        from mlem.core.requirements import MODULE_PACKAGE_MAPPING
-
-        return [MODULE_PACKAGE_MAPPING.get(r, r) for r in self.reqs]
+        # since this must work without these packages installed,
+        # we use mapping for cases when module_name != package_name
+        module_package = {
```

Comment:
maybe just return the mapping constant for this one?

Reply:
Sure

Reply:
but, on other hand, this is a single place where it's used. I can make a constant, but what's a point?